Makefile

```diff
@@ -55,6 +55,8 @@ style-check:
 style:
 	@rustup component add rustfmt 2> /dev/null
 	./scripts/style.sh
+	# enabled as soon as all people get this one
+	#nixfmt $(find . -name "*.nix" -type f | tr "\n" " ")
```

Comment:
why is this commented?

Reply:
because bot all who uses this already have nixfmt. will wait for larger nix adoption.

Reply:
as soon as there will be no complains nix not woking. will on this by default